src/sentry/models/event.py

```diff
@@ -535,6 +512,26 @@ def id(self):
         # have to reference the row id anyway.
         return self.event_id
 
+    @property
+    def timestamp(self):
+        return self.snuba_data["timestamp"]
```

Comment:
SnubaEvent cannot be populated through a query on the transactions dataset today, right ?

Reply:
No, this will need to change but SnubaEvent always comes from the events dataset even if it is a transaction.